### ISSUE TITLE
Use established software package repositories (bioconductor, bioconda) and provide comprehensive documentation (including vignettes/tutorials)

### ISSUE THREAD

**Issue by ceiron61** [carpenterlab/open-science-rules#14]
The bioinformatics community has assembled some great software package repositories such as Bioconductor and Bioconda (and probably others I'm not aware of).
Scientists who create new tools should try, where possible, to release them as part of these framework to increase ease of access, interoperability and benchmarking.

Related to this, software documentation is of paramount importance. Aside from the obvious requirement of a manual, developers should also aim to provide vignettes/tutorials for the most common use cases. IIRC this is enforced in submissions to Bioconductor.

**Comment by gwaybio** [comment]
Yeah this is great. I think of this rule as under some sort of reproducibility umbrella. 

Open science is only as good as the "science" part - and good science requires reproducibile practices!

**Comment by gwaybio** [comment]
We can also use this rule to link out to other reproducibility papers that will cover this topic comprehensively.

**Comment by gwaybio** [comment]
A couple related blogposts mentioned in #11 - essentially, open science practice follows from open data repositories with effective annotations and computational tools with reproducible workflows - all pursued in a version controlled environment
